allennlp/predictors/predictor.py

```diff
@@ -96,6 +179,16 @@ def predict_instance(self, instance: Instance) -> JsonDict:
         outputs = self._model.forward_on_instance(instance)
         return sanitize(outputs)
 
+    def predictions_to_labeled_instances(self,
+                                         instance: Instance,
+                                         outputs: Dict[str, np.ndarray]) -> List[Instance]: \
+                                         # pylint: disable=unused-argument,no-self-use
+        """
+        Adds labels to the :class:`~allennlp.data.instance.Instance`s passed in and returns it.
+        """
+        raise RuntimeError("you need to implement this method if you \
```

Comment:
You can just use string continuations here.  End the string, remove the backslash, start a new string on the next line.  Contiguous strings get concatenated in python.